docs/guides/make_bootable_media.md

```diff
@@ -0,0 +1,24 @@
+# Make Bootable Media
+How to image a drive using your Mark II's "Make Bootable Media" skill.
+
+## Requires
+Two USB drives, or a USB drive and an SSD drive, or two SSD drives with supplemental power to one of them. Any data on the secondary drive will be erased by this process. 
+
+## Steps
+1. Start with your machine booted up, with the boot drive in the upper left USB port.
```

Comment:
Same note as above; also, if the device is already running the port is probably extraneous information here

Reply:
Would it succeed at making new bootable media with the machine running off the right-hand port, and the blank drive in the left-hand port?

Reply:
yes; it will write to the second initialized storage device (the first being what was booted to)